### ISSUE TITLE
logdir setting has no effect

### ISSUE THREAD

**Issue by AwePhD** [microsoft/vscode-tensorboard#18]
The setting of the log dir is not taken into account in the extension. Namely, setting the tensorboard log dir does not disable the prompt.

In the [related commit](https://github.com/microsoft/vscode-tensorboard/pull/7/commits/fa3b2910897822caf85fca293c19aeb94bcd8393#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519), in `package.json`, we set this setting to be `tensorBoard`. But in the actual `src/configuration.ts`, we search a `tensorboard`. I guess if we uppercase the B on [this line](https://github.com/microsoft/vscode-tensorboard/blob/311f4d56cd1ec874e8b560a759c71eb43995d824/src/configuration.ts#L95), it would fix the problem? (I am not a JS dev, so I cannot be sure at all)

Current workaround, lowercase the B in the JSON config manually. 
From:
```json
"tensorBoard.logDirectory": "/path/to/log/dir"
```
To:
```json
"tensorboard.logDirectory": "/path/to/log/dir"
```

**Comment by YoniChechik** [comment]
@DonJayamanne thanks for this extansion!
as of today this fix still doesn't work (on pre-release as well)

this workaround worked for me:
> Current workaround, lowercase the B in the JSON config manually. From:
> 
> ```json
> "tensorBoard.logDirectory": "/path/to/log/dir"
> ```
> 
> To:
> 
> ```json
> "tensorboard.logDirectory": "/path/to/log/dir"
> ```

**Comment by laclouis5** [comment]
The issue does not seem to be fixed in the last version (`v2023.10.1002992421`). Do any maintainer have an idea on when this fix will be delivered? Thanks for your work!
